docs/adding_tracks.rst

```diff
@@ -530,26 +527,26 @@ Finally, specify the compressed file name in your ``track.json`` file in the ``s
 
 Specifying ``compressed-bytes`` (file size of ``documents.json.bz2``) and ``uncompressed-bytes`` (file size of ``documents.json``) is optional but helps Rally to provide progress indicators and also verify integrity.
 
-You've now mastered the basics of track development for Rally. It's time to pat yourself on the back before you dive into the advanced topics!
+You've now mastered the basics of track development for Rally. It's time to pat yourself on the back before you dive into the advanced topics.
```

Comment:
I must say I liked the use of an exclamation mark here! I know https://developers.google.com/style/exclamation-points discourages it, but we are not calling attention to an important point, but rather, lighten the mood and "reward" the reader, after digging into a deep topic.

Reply:
I reverted my change.